src/styles/blocks/page.css

```diff
@@ -0,0 +1,9 @@
+.page {
+  margin: 0;
+  font-family: Avenir, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could use 'font-family: Avenir, Arial, sans-serif;' to ensure that if Avenir is not available, the browser can use Arial or a default sans-serif font.